apis/fluentd/v1alpha1/plugins/output/cloudwatch.go

```diff
@@ -0,0 +1,95 @@
+package output
+
+// Cloudwatch defines the parametes for out_cloudwatch output plugin
+type Cloudwatch struct {
```

Comment:
I think we should change `Cloudwatch` to `CloudWatch` following the official name:
![image](https://user-images.githubusercontent.com/18525465/220831671-f0362ccb-a842-4919-a77f-7bdbd8a2ab4f.png)

Reply:
Agreed.  I'll make the change.